composer.json

```diff
@@ -15,7 +15,8 @@
         "doctrine/data-fixtures": "1.*@alpha",
         "doctrine/doctrine-fixtures-bundle": "2.*@alpha",
         "doctrine/orm": "2.*",
-        "doctrine/doctrine-bundle": "*"
+        "doctrine/doctrine-bundle": "*",
+        "symfony/property-access": "~2.3"
```

Comment:
Isn't this going to block you on the 2.3 branch where we'd actually probably want 2.* ?